csrankings.csv

```diff
@@ -5751,6 +5753,7 @@ Hongdong Li,Australian National University,http://users.cecs.anu.edu.au/~hongdon
 Hongfei Yan,Peking University,http://eecs.pku.edu.cn/EN/People/Faculty/Detail/?ID=6010,NOSCHOLARPAGE
 Hongfeng Yu,University of Nebraska,http://vis.unl.edu/~yu,yeWsFW4AAAAJ
 Hongjie Chen,Peking University,http://www.sei.pku.edu.cn/people/chenhj,NOSCHOLARPAGE
+Honglak Lee,University of Michigan,http://eecs.umich.edu/~honglak,fmSHtE8AAAAJ

```

Comment:
Apparently at Google Brain -- only full-time faculty are eligible for inclusion.

Reply:
Thank you for your review!
Honglak Lee is currently an associate professor in UofM. He's teaching a class here.